Web/src/components/games/PandaSequence/useSeqGenerator.ts

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import seedrandom from 'seedrandom';
+import { Sequence } from './Sequence';
+
+const INITIAL_LENGTH = 5; // length of each memory sequence
+const INITIAL_TIMESTEP = 500; // duration of each timestep
+const LENGTH_FACTOR = 1.2;
+const TIMESTEP_FACTOR = 0.8;
+
+function* randomWithinBounds(seed:string, lowerBound:number, upperBound:number)
+  :Generator<number, number, void> {
+  const random = seedrandom(seed);
+  while (true) {
+    const x = Math.floor(random() * (upperBound - lowerBound) + lowerBound);
+    yield x;
+  }
+}
+
+/**
+ * Takes in a seed and returns randomly generated Sequences of numbers between
+ * the range of lowerBound and upperBound.
+ */
+export default function UseSeqGenerator(
+  seed = Date.now().toString(),
+  lowerBound = 1,
+  upperBound = 5,
+) {
+  const [generator] = useState(randomWithinBounds(seed, lowerBound, upperBound));
+  const [count, setCount] = useState(0);
+  const [sequences, setSequences] = useState(Array<Sequence>());
```

Comment:
```suggestion
  const [sequences, setSequences] = useState<Sequence[]>([]);
```

This is usually what I do. @juxd @hzxcaryn thoughts?

Reply:
Oh same here!

Reply:
This part is removed because I updated it to store a reference to a sequence, rather than an array of sequences. I'll take note of the standardised syntax in future scenarios :)